GliaWidgets/SecureConversations/Welcome/Theme+SecureConversationsWelcome.swift

```diff
@@ -39,7 +39,7 @@ extension Theme {
         let messageTitleStyle = SecureConversations.WelcomeStyle.MessageTitleStyle(
             title: Localization.MessageCenter.Welcome.messageTitle,
             font: font.mediumSubtitle1,
```

Comment:
I guess it would correct to change font to `mediumSubtitle2` to keep resolved font the same? Does it make sense?

Reply:
The same for Survey

Reply:
Oh, if it matches with design, then it's expected. Thanks for explaining